dietpi/dietpi-software

```diff
@@ -12465,6 +12484,47 @@ _EOF_
 
 		fi
 
+    software_id=181 # Paper
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			# Make sure user agrees to the EULA
+			G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+			if G_WHIP_YESNO "Do you agree to the Minecraft EULA found at.
+\nhttps://account.mojang.com/documents/minecraft_eula"; then
```

Comment:
When user selects "no" here, I it will be asked again when accessing the server the first time, right?